docs/bodyswap.rst

```diff
@@ -14,15 +14,27 @@ Usage
 ::
 
     bodyswap [--unit <id>]
+    bodyswap [--linger]
```

Comment:
I prefer the form you wrote in the docs below (and in the [tombstone entry](https://github.com/DFHack/dfhack/pull/4781/files)): `bodyswap linger`, not `bodyswap --linger`

in that case, you should write the usage as `bodyswap linger`, not `bodyswap [linger]`, since the zero option case is already covered by the first usage line

Reply:
Oh crap yah I prefer bodyswap linger too, I'm just dumb about the arguments and dunno how to handle it properly